website/registrations/services.py

```diff
@@ -264,8 +265,14 @@ def _create_member_from_registration(registration: Registration) -> Member:
         birthday=registration.birthday,
         show_birthday=registration.optin_birthday,
         receive_optin=registration.optin_mailinglist,
+        emergency_contact=registration.emergency_contact,
+        emergency_contact_phone_number=registration.emergency_contact_phone_number,
+        photo=registration.photo,  # TODO change the image path to the correct directory and verify thumbnails
```

Comment:
You don't need to verify thumbnails, those are created on the first time a specific size of the image is requested.